bucket/megacmd.json

```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://mega.nz/cmd",
+    "description": "Command line tool to access MEGA(mega.nz)",
+    "version": "1.1.0",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/meganz/MEGAcmd/blob/master/build/installer/terms.txt"
+    },
+    "url": "https://mega.nz/MEGAcmdSetup.exe#/dl.7z",
+    "hash": "9a7ddc65cc6a792af2d5f54d69b2e08a5a48db050c8d3131011805b508a2f20d",
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Recurse",
+            "Get-ChildItem \"$dir\" 'mega-*.bat' | ForEach-Object {",
```

Comment:
Just add them all as bin. What's problem?

Reply:
Doing this manually is nonsense and dangerous.

Reply:
`gci . 'mega-*.bat' | Select -expand Basename | % { "\"$_\"," } | clip` and paste it into manifest

Reply:
There may be more batch file added in the future. 
If I simply put every thing in `bin`, new commands will be ignored. 
(Until some user found out and file an issue)

Reply:
And? That is casual.

Reply:
You mean just add `gci...` to the install script? Or replacing the whole part after `Get-ChildItem`?

Reply:
No ...

1. Open manifest inside text editor
1. Create bin property as array.
1. Execute `gci . 'mega-*.bat' | Select -expand Basename | % { "\"$_\"," } | clip` inside terminal
1. Press CTRL+V in bin property inside manifest
1. Remove trailing comma
1. Press shortcut for formatting json file
1. Commit changes

Reply:
Thanks for the instruction.
I found that batch files in `bin` will not work without adding `.bat` in the end.